cmd/migrations/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"database/sql"
+	"flag"
+	"fmt"
+
+	"github.com/go-devs-ua/octagon/app/repository/pg"
+	"github.com/go-devs-ua/octagon/cfg"
+	"github.com/go-devs-ua/octagon/lgr"
+	_ "github.com/lib/pq"
+
+	migrate "github.com/rubenv/sql-migrate"
+)
+
+func main() {
+	logger := lgr.New()
+
+	opt, err := cfg.GetConfig()
+	if err != nil {
+		logger.Errorf("Failed to get config from .env: %+v\n", err)
+		return
+	}
+
+	db, err := pg.ConnectDB(opt)
+	if err != nil {
+		logger.Errorf("%+v\n", err)
+		return
+	}
+
+	direction := flag.String("migrate", "", "applying migrations 'up/down'")
+	flag.Parse()
+
+	if *direction == "up" || *direction == "down" {
```

Comment:
```go
	if *direction == "up" || *direction == "down" {
		if err = migrateDB(db, logger, *direction); err != nil {
			logger.Errorf("Failed making migrations: %v\n", err)
		}
	} else {
		fmt.Println("wrong flag provided, choose '-migrate up' or '-migrate down'")
		return
	}
```

I think this version is a bit more readable. Keep in mind, that happy path should go through the function and exceptions (errors etc) should be in if statements. 
You can't always achieve this but we have try to doing this when it's possible. 
```go
	if *direction != "up" && *direction != "down" {
		fmt.Println("wrong flag provided, choose '-migrate up' or '-migrate down'")
		return
	} 
	
        if err := migrateDB(db, logger, *direction); err != nil {
		logger.Errorf("Failed making migrations: %v\n", err)
	}
```

Reply:
Fixed.